src/sage/rings/polynomial/polynomial_ring.py

```diff
@@ -844,9 +846,9 @@ def _magma_init_(self, magma):
             Univariate Polynomial Ring in y over Rational Field
             sage: S.1                                       # optional - magma
             y
-            sage: magma(PolynomialRing(GF(7), 'x'))         # optional - magma          # optional - sage.rings.finite_rings
+            sage: magma(PolynomialRing(GF(7), 'x'))     # optional - magma              # needs sage.rings.finite_rings
             Univariate Polynomial Ring in x over GF(7)
-            sage: magma(PolynomialRing(GF(49,'a'), 'x'))    # optional - magma          # optional - sage.rings.finite_rings
+            sage: magma(PolynomialRing(GF(49,'a'), 'x'))        # optional - magma, needs sage.rings.finite_rings
             Univariate Polynomial Ring in x over GF(7^2)
```

Comment:
Is it intentional that you use different styles in L849 and L851?

Reply:
the fixer switches to comma style when space is tight

I'll change this one by using a block-scoped tag for `# optional - magma`

Reply:
done in 2625d2af7e